src/promptflow/sample_trace.json

```diff
@@ -0,0 +1,290 @@
+[
+    {
+        "name": "hello",
+        "context": {
+            "trace_id": "0xa0f1ba62b17f09c0cc6fb6a176ed6b77",
+            "span_id": "0x05df0e6a82ee2127",
+            "trace_state": "[]"
+        },
+        "kind": "SpanKind.INTERNAL",
+        "parent_id": "0xa9bf586cf9fba31d",
+        "start_time": "2024-01-22T14:26:57.338927Z",
+        "end_time": "2024-01-22T14:26:57.339127Z",
+        "status": {
+            "status_code": "OK"
```

Comment:
I think in OpenTelemetry design this status_code field is optional? If not present, it means OK?